src/test/scala/nl.knaw.dans.easy.download/ServletSpec.scala

```diff
@@ -53,8 +53,8 @@ class ServletSpec extends TestSupportFixture with EmbeddedJettyContainer
     (app.http.copyHttpStream(_: URI)) expects new URI(s"http://localhost:20110/bags/$uuid/${ path.escapePath }") repeat n
   }
 
-  private def expectAuthorisation(path: Path) = {
-    (app.http.getHttpAsString(_: URI)) expects new URI(s"http://localhost:20170/$uuid/${ path.escapePath }") anyNumberOfTimes
+  private def expectAuthorisation(path: Path, n: Int = 1) = {
```

Comment:
I presume you want to make it more similar to `expectAuthentication` by adding the parameter `n`. And I get it that is has a default value. Still ... a bit of a slippery slope to [YAGNI-land](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it).

Reply:
Believe it or not, but it's not even that... It's more of a "_I forgot to take it out again after I realized I didn't need it, because something else caught my attention_" 😜